app/components/global/AppBar/AppBar.js

```diff
@@ -46,24 +46,28 @@ const DesktopView = styled(Flex)`
   `};
 `
 
-const AppBar = ({ user }) => (
-  <AppBarContainer>
-    <BurgerMenu menuItems={user && [{ label: 'Dashboard', link: '/' }]} />
-    <LogoLink href="https://elifesciences.org">
-      <img alt="eLife" src="/assets/elife-logo.png" />
-    </LogoLink>
-    <DesktopView>
-      <Separator />
-      <Box flex="1 1 auto">
-        {user && (
-          <AppBarLink exact to="/">
-            Dashboard
-          </AppBarLink>
-        )}
-      </Box>
-    </DesktopView>
-    <ProfileMenu user={user} />
-  </AppBarContainer>
-)
+const AppBar = ({ user, userMenuItems, defaultMenuItems }) => {
+  const menuItems = [...(user ? userMenuItems : []), ...defaultMenuItems]
```

Comment:
Makes more sense to default `userMenuItems` in the params, i.e.
`const AppBar = ({ user, userMenuItems = [], defaultMenuItems }) => { ... }`

Reply:
This wouldn't work with how the logic is currently as that would require me changing 
```suggestion
  const menuItems = [...(user && userMenuItems), ...defaultMenuItems]
```
which would break when the user isn't signed in.

Reply:
Right, I would suggest changing it to 
```suggestion
  const menuItems = [...(loggedIn ? userMenuItems : []), ...defaultMenuItems]
```